src/models/db/account.js

```diff
@@ -69,6 +67,8 @@ class Account extends Model {
   }
 
   static convertFromPersistent (data) {
+    data.is_disabled = Boolean(data.is_disabled)
```

Comment:
Yeah, I'll add a comment. Some databases store booleans as `0` and `1` so it has to be converted back. I probably shouldn't convert to number in `convertToPersistent`, I'll try changing that.